### DISCUSSION TITLE
error TS2688: Cannot find type definition file for 'responselike'.

### DISCUSSION THREAD

**Discussion by zombieJ** [DefinitelyTyped/DefinitelyTyped#62115] [Issues with a @types package]
Seems v3 do not need ts define anymore but it breaks prev version:

ref PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62102

CI Error: https://github.com/react-component/util/runs/8185103317?check_suite_focus=true

[a comment by an automated account is not shown]

**Comment by tristen** [comment]
@zombieJ see https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/62111#discussioncomment-3557529
